README.md

```diff
@@ -34,6 +34,9 @@ Check out [the docs](http://hexdocs.pm/ex_machina/ExMachina.html) for more detai
 Define factories:
 
 ```elixir
+# You could put this in `test/support/factory.ex` if you use it for tests only,
```

Comment:
@henrik What if you do `test/factory.exs`. I think it might beed to be a scripted elixir file. Not sure though